### PR TITLE
Optimized debug output in CM about interfaces when switching controllers.

### PR DESCRIPTION
**Pull request by destogl**
A part of #1237

### CODE REVIEW

File: controller_manager/src/controller_manager.cpp [hunk 1]
```diff
@@ -1222,8 +1223,19 @@ controller_interface::return_type ControllerManager::switch_controller(
...
+    RCLCPP_DEBUG(get_logger(), " - %s", interface.c_str());
+  }
+  RCLCPP_DEBUG(get_logger(), "Request for command interfaces from deactivating controllers:");
+  for (const auto & interface : deactivate_command_interface_request_)
+  {
+    RCLCPP_DEBUG(get_logger(), " - %s", interface.c_str());
+  }
+
+  if (!resource_manager_->prepare_command_mode_switch(
+        activate_command_interface_request_, deactivate_command_interface_request_))
```
Comment (destogl) [inline controller_manager/src/controller_manager.cpp:1238]:
```suggestion
  if (
    !activate_command_interface_request_.empty() || !deactivate_command_interface_request_.empty())
```

**Review by bmagyar — DISMISSED** [review]
Beautiful!

File: controller_manager/src/controller_manager.cpp [hunk 2]
```diff
@@ -1222,6 +1223,17 @@ controller_interface::return_type ControllerManager::switch_controller(
     return controller_interface::return_type::OK;
   }
 
+  RCLCPP_DEBUG(get_logger(), "Request for command interfaces from activating controllers:");
+  for (const auto & interface : activate_command_interface_request_)
+  {
+    RCLCPP_DEBUG(get_logger(), " - %s", interface.c_str());
+  }
+  RCLCPP_DEBUG(get_logger(), "Request for command interfaces from deactivating controllers:");
```
Comment (saikishor) [inline controller_manager/src/controller_manager.cpp:1231]:
```suggestion
  RCLCPP_DEBUG(get_logger(), "Release of command interfaces from deactivating controllers:");
```

A minor suggestion

**Review by saikishor — APPROVED** [review]
LGTM!

**Pull request merged**